plugin/plugin_context.go

```diff
@@ -147,27 +132,6 @@ func (c *pluginContext) PluginConfig() PluginConfig {
 	return c.pluginConfig
 }
 
-func (c *pluginContext) RefreshIAMToken() (string, error) {
```

Comment:
Removing the definition _shouldn't_ be a breaking change since it's defined in repository.go

Reply:
This could be a problem/break for plug-in teams as they don't use the `configuration/core_config` package. Developers are instructed to use PluginContext basically for their access / knowledge of core CLI config, etc. For example for IAM Tokens it shows as this:  https://test.cloud.ibm.com/docs/cloud-cli-plugin-sdk?topic=cloud-cli-plugin-sdk-gen-cli . I would think if the sample plug-in `plugin_examples/context.go` works properly with this SDK change then all should be fine for plug-in teams.

Reply:
It should still work since the method is included in the `PluginContext` interface even though it's defined in `repository.go`. I tested a local copy of `plugin_examples/context.go` and called `context.RefreshToken()`

```go
func (p *PrintContext) Run(context plugin.PluginContext, args []string) {
	table := p.ui.Table([]string{"Name", "Value"})
	token, _ := context.RefreshIAMToken()
	table.Add("token", token)
	table.Print()
}
```

output:
```
jrangel:ibm-cloud-cli-sdk jrangel$ ic context
Name    Value   
token   Bearer (redacted)
```

Reply:
Per our convo on slack, no more concerns here.